query-sql/src/lib.rs

```diff
@@ -172,6 +172,19 @@ impl QueryFragment for Op {
     }
 }
 
+macro_rules! interpose {
```

Comment:
Blech.  At least comment on what this does, for the next poor sod.  You already have a `QueryFragment` and I see that source aliases don't implement it 'cuz of ownership.  Consider newtyping `SourceAlias` so that can impl `QF` -- you might not need this macro.